episodes/vaccine-comparisons.Rmd

```diff
@@ -0,0 +1,968 @@
+  time_end = matrix(40 + 150, nrow(contact_matrix)),
+  nu = matrix(c(0, 0, 0.001 * 3))
+)
+
+output_vaccinate_group_1 <- model_default(
+  # population
+  population = uk_population,
+  # rate
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  # intervention
+  vaccination = vaccinate_group_1,
+  # time
+  time_end = 200, increment = 1.0
+)
+
+output_vaccinate_group_2 <- model_default(
+  # population
+  population = uk_population,
+  # rate
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  # intervention
+  vaccination = vaccinate_group_2,
+  # time
+  time_end = 200, increment = 1.0
+)
+
+
+output_vaccinate_group_3 <- model_default(
+  # population
+  population = uk_population,
+  # rate
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  # intervention
+  vaccination = vaccinate_group_3,
+  # time
+  time_end = 200, increment = 1.0
+)
+
+vaccinate_group_1_infections <- new_infections(output_vaccinate_group_1,
+                                               compartments_from_susceptible =
+                                                 "vaccinated", by_group = FALSE)
+vaccinate_group_2_infections <- new_infections(output_vaccinate_group_2,
+                                               compartments_from_susceptible =
+                                                 "vaccinated", by_group = FALSE)
+vaccinate_group_3_infections <- new_infections(output_vaccinate_group_3,
+                                               compartments_from_susceptible =
+                                                 "vaccinated", by_group = FALSE)
+
+vaccinate_group_1_infections$intervention_type  <- "vaccinate group 1"
+vaccinate_group_2_infections$intervention_type  <- "vaccinate group 2"
+vaccinate_group_3_infections$intervention_type  <- "vaccinate group 3"
+
+output_infections <- rbind(baseline_infections,
+                           vaccinate_01_infections,
+                           vaccinate_group_1_infections,
+                           vaccinate_group_2_infections,
+                           vaccinate_group_3_infections)
+
+
+output_infections %>%
+  ggplot() +
+  geom_line(
+            aes(time, new_infections, colour = intervention_type,
+              linetype = intervention_type
+            ),
+            linewidth = 1) +
+  scale_y_continuous(
+    labels = scales::comma
+  ) +
+  geom_vline(
+    xintercept = c(
+      vaccinate_01$time_begin,
+      vaccinate_01$time_end
+    ),
+    linetype = 2
+  ) +
+  theme_bw() +
+  labs(
+    x = "Simulation time (days)",
+    y = "Individuals"
+  )
+```
+
+
+:::
+
+Vaccinating group 3 only results in the highest epidemic peak size. Targeting group 2 results in the smallest epidemic peak size. Whereas targeting all age groups pushes the epidemic peak time later.  
```

Comment:
Worth briefly mentioning (either here or referencing elsewhere if already covered in another episode) that we're often interested in both the peak size (i.e. healthcare pressure at single point in time) as well as overall epidemic size (i.e. cumulative disease)?